server/src/server.ts

```diff
@@ -41,7 +41,11 @@ async function startApp() {
     //server.use(cookieParser());
 
     // setup sane defaults for CORS and HTTP headers
-    server.use(helmet());
+    server.use(
+      helmet({
+        contentSecurityPolicy: false,
```

Comment:
Det her er for å kunne kjøre inline-scripts, som react krever. sånn sett ikke noe nytt, men "helmet" pakken har i en oppdatert versjon lagt på en header som hindrer det by default.